### ISSUE TITLE
Chapter 5 bug - no vocab error

### ISSUE THREAD

**Issue by ghost** [falloutdurham/beginners-pytorch-deep-learning#2]
The notebook for chapter 5 needs a line like the following:

LABEL.build_vocab(train, max_size=2)

Otherwise, when you run the train method there's a no vocab error that I don't have handy just now.

**Comment by ryanbrand** [comment]
I'm having the same issue and can't find a suitable workaround anywhere.  Which versions of pytorch/torchtext are assumed for this notebook?

**Comment by falloutdurham** [comment]
LABEL.build_vocab(train, max_size=2) (following the other vocab build) should fix it, as suggested.

torchtext is 0.2.0, pytorch is 1.1.0
